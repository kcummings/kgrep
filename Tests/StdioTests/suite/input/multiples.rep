This~There
and~but
line~source