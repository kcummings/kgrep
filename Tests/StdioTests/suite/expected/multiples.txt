There is a test
of source one
but source two
but the final source by source.
